package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "4.11.15",
+  "version": "4.11.16",
```

Comment:
After doing this, please do a `npm i`, and also commit the package lock file.

(The package lock file specifies the exact versions of all packages used to build the app. We commit this so that Github builds the app with the exact same dependencies as locally.)